config/webpack.bundle.js

```diff
@@ -17,7 +17,8 @@ const CleanWebpackPlugin = require('clean-webpack-plugin')
  */
 module.exports = merge(require('./webpack.config.js'), {
   entry: {
-    'aepp.base': './src/styles/index.scss',
+    'aepp.global': './src/styles/index.scss',
+    'aepp.fonts': './src/styles/_fonts.scss',
```

Comment:
Do you have fonts.scss named with `_` somewhere?

Reply:
@davidyuk yep, mistake. I've fixed it.